contacts/main.js

```diff
@@ -0,0 +1,253 @@
+ /*
+description or something
+*/
+
+
+// person entity
+class Person {
+  constructor(id, firstName, lastName, image, likes = 0, dislikes = 0){
+    this.id = id;
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.image = image;
+    this.likes = likes;
+    this.dislikes = dislikes;
+  }
+  getFullName() {
+    return this.id + ' ' + this.firstName + ' ' + this.lastName;
+  }
+}
+
+
+function getListOfContacts(){
+  let array = [];
+  var id;
+  var person;
+
+  id = generateUniqueID();
+  person = new Person(id, 'Alin', 'Ionescu', 'images/profile-pics/pic1.jfif');
+  array.push(person);
+  id = generateUniqueID();
+  person = new Person(id, 'Leona', 'Sherman', 'images/profile-pics/pic2.jfif');
+  array.push(person);
+  id = generateUniqueID();
+  person = new Person(id, 'Annette', 'Carlson', 'images/profile-pics/pic3.jfif');
+  array.push(person);
+
+  return array;
+}
+
+function populateContactsList(){
+  let arr = getListOfContacts();
+  for(let i=0; i<arr.length; i++){
+    console.log(arr[i].getFullName());
+    addToHTMLNewPerson(arr[i]);
+    attachConections(arr[i]);
+  }
+}
+//-----------------------------actions----------------------------------
+// function that hgas multiple actions
+function deleteAllButtonAction() {
+  // delete selected contacts
+  deleteSelectedContacts();
```

Comment:
you can use just one of the functions, there is no point in having a method, that calls another method, when they are on the same level

Reply:
in the function deleteAllButtonAction() will be called another function that will refresh the favs list